ktor-swagger-ui/src/main/kotlin/io/github/smiley4/ktorswaggerui/builder/example/ExampleContextImpl.kt

```diff
@@ -34,7 +29,7 @@ class ExampleContextImpl : ExampleContext {
      */
     fun add(routes: Collection<RouteMeta>) {
         collectExampleDescriptors(routes).forEach { exampleDescriptor ->
-            rootExamples[exampleDescriptor] = generateExample(exampleDescriptor)
+            rootExamples[exampleDescriptor] = exampleDescriptor
```

Comment:
i think i am missing why we need to generate the example in the getter and not here. Couldn't `collectExampleDescriptors` return the descriptor together with a type so that we can generate the final swagger Example object here?

Reply:
did the thing